add osu! Battle Archive Malaysia Tournament Article (#6706)
ppy/osu-wiki: wiki/Tournaments/o!BAM/1/en.md

---
tags:
  - o!BAM
---

# osu! Battle Archive Malaysia

![o!BAM banner](img/banner.png) 

> The **osu! Battle Archive Malaysia** (***o!BAM***) is a was a Malaysian double-elimination 2v2 osu! tournament hosted by ![][flag_MY] [LouisZpro](https://osu.ppy.sh/users/6231292) and ![][flag_MY] [Tzero](https://osu.ppy.sh/users/6088976) This was the first instalment of the osu! Battle Archive Malaysia.

Reviewer: ```suggestion
The **osu! Battle Archive Malaysia** (***o!BAM***) was a Malaysian double-elimination 2v2 osu! tournament hosted by ![][flag_MY] [LouisZpro](https://osu.ppy.sh/users/6231292) and ![][flag_MY] [Tzero](https://osu.ppy.sh/users/6088976). It was the first instalment of the osu! Battle Archive Malaysia.
```

## Tournament schedule

| Event | Timestamp |
| --: | :-- |
| Registration phase | 2021-08-10/2021-08-29 |

> | Qualifiers | 2021-09-04/2021-09-05

Reviewer 2: ```suggestion
| Qualifiers | 2021-09-04/2021-09-05 |
```

> | Live drawings | 2021-09-06 (21:00 UTC+8) |

Reviewer 2: Change this to UTC date and time.

| Round of 32 | 2021-09-11/2021-09-12 |
| Round of 16 | 2021-09-18/2021-09-19 |
| Quarterfinals | 2021-09-25/2021-09-26 |

> | Semifinals | 2021-10-2/2021-10-3 |

Reviewer: ```suggestion
| Semifinals | 2021-10-02/2021-10-03 |
```

> | Finals week 1 | 2021-10-9/2021-10-10 |

Reviewer: ```suggestion
| Finals (week 1) | 2021-10-09/2021-10-10 |
```

> | Finals week 2 | 2021-10-16/2021-10-17 |

Reviewer: ```suggestion
| Finals (week 2) | 2021-10-16/2021-10-17 |
```

## Prizes

| Placing | Prize(s) |
| :-: | :-- |

> | ![Gold crown](/wiki/shared/crown-gold.png "1st place") | 2 months supporter per player,Animated Profile Banners, Exclusive Wrist-rest custom-designed by Wristlabs.net  |

Reviewer: ```suggestion
| ![Gold crown](/wiki/shared/crown-gold.png "1st place") | 2 months of osu!supporter per each team member, animated profile banners, exclusive wrist-rest custom-designed by [Wristlabs.net](https://wristlabs.net)  |
```

> | ![Silver crown](/wiki/shared/crown-silver.png "2nd place") | 1 months supporter per player,Animated Profile Banners |

Reviewer: ```suggestion
| ![Silver crown](/wiki/shared/crown-silver.png "2nd place") | 1 month of osu!supporter per each team member, animated profile banners |
```

> | ![Bronze crown](/wiki/shared/crown-bronze.png "3rd place") | 1 months supporter per player,Animated Profile Banners |

Reviewer: ```suggestion
| ![Bronze crown](/wiki/shared/crown-bronze.png "3rd place") | 1 month of osu!supporter per each team member, animated profile banners |
```

## Organisation

> The osu! Battle Archive Malaysia is run by various community members.

Reviewer: ```suggestion
The osu! Battle Archive Malaysia was run by various community members.
```

| Administrator | ![][flag_MY] [Laphii](https://osu.ppy.sh/users/9364594), ![][flag_SG] [Ayucchi](https://osu.ppy.sh/users/7823498) |
| Mappool selector | ![][flag_MY] [LouisZpro](https://osu.ppy.sh/users/6231292), ![][flag_MY] [Tzero](https://osu.ppy.sh/users/6088976), ![][flag_SG] [Ayucchi](https://osu.ppy.sh/users/7823498) |
| Streamer | ![][flag_MY] [Laphii](https://osu.ppy.sh/users/9364594), ![][flag_MY] [Hand Sanitiser](https://osu.ppy.sh/users/5091293), ![][flag_MY] [Koyaiba](https://osu.ppy.sh/users/7062729) | 
| Commentator | ![][flag_MY] [MyAngelFable](https://osu.ppy.sh/users/11234664), ![][flag_MY] [Malaidan](https://osu.ppy.sh/users/14279913), ![][flag_MY] [fancyyy](https://osu.ppy.sh/users/12616268), ![][flag_MY] [sealw4ll](https://osu.ppy.sh/users/12289818), ![][flag_MY] [Banner](https://osu.ppy.sh/users/14290988), ![][flag_MY] [MinaMyoui](https://osu.ppy.sh/users/6996926), ![][flag_MY] [IHateAutoNicks](https://osu.ppy.sh/users/10758397), ![][flag_MY] [Chiyuu](https://osu.ppy.sh/users/8226107), ![][flag_MY] [Its not Marvin](https://osu.ppy.sh/users/13600612) |

> | Referee | ![][flag_MY] [Xyrelyx](https://osu.ppy.sh/users/3765632), ![][flag_MY] [Laphii](https://osu.ppy.sh/users/9364594),![][flag_VN] [rock-on](https://osu.ppy.sh/users/9676089), ![][flag_MY] [Vexine](https://osu.ppy.sh/users/4042706), ![][flag_FR] [Taevas](https://osu.ppy.sh/users/7276846), ![][flag_FR] [DaNoobSlayaar](https://osu.ppy.sh/users/8688777), ![][flag_FR] [Wen4ng3L](https://osu.ppy.sh/users/9288833), ![][flag_MY] [Hand Sanitiser](https://osu.ppy.sh/users/5091293), ![][flag_MY] [MinaMyoui](https://osu.ppy.sh/users/6996926) |

Reviewer: as someone who handles the wiki page of [the currently ongoing oseat5](http://osu.ppy.sh/wiki/id/Tournaments/oSEAT/5) i'm quite sure that DaNoobSlayaar isn't from France lol

Author: I seen quite a lot of error because I think I dint hit the save button when i was commiting

| Graphic Designer | ![][flag_MY] [LouisZpro](https://osu.ppy.sh/users/6231292), ![][flag_MY] [Laphii](https://osu.ppy.sh/users/9364594) |
| Spreadsheet manager | ![][flag_MY] [Laphii](https://osu.ppy.sh/users/9364594), ![][flag_MY] [Tzero](https://osu.ppy.sh/users/6088976)
| Statistician | ![][flag_MY] [Laphii](https://osu.ppy.sh/users/9364594), ![][flag_MY] [Khoo Hao Yit](https://osu.ppy.sh/users/9534110) |
| Wiki editor | ![][flag_MY] [Laphii](https://osu.ppy.sh/users/9364594) |

## Links

- **[Main spreadsheet](https://docs.google.com/spreadsheets/d/1FR4FDh9655BF6d7d275BAQDJX3jT7o4wVcJb8AYdBec/edit#gid=0)**
- [Forum thread](https://osu.ppy.sh/community/forums/topics/1393142)
- [Discord server](https://discord.gg/GYqWsQbEbF)
- [Challonge brackets](https://challonge.com/oBAM)
- [Livestream channel](https://www.twitch.tv/osumalaysia)
- **[Statistics sheet](https://docs.google.com/spreadsheets/d/1GY_On5qdet-PYSYbfMEqOJq780CTUKxDHdcf9nJDjbQ/edit#gid=1947051038)**

## Participants

| Team | Members |
| :-- | :-- |

> | **nomicstreamers** | ![][flag_MY] **[DuoX](https://osu.ppy.sh/users/9560694)**, ![][flag_SG] [ShaneLiang](https://osu.ppy.sh/users/6716499) |

Reviewer: ```suggestion
| **nomicstreamers** | ![][flag_MY] **[DuoX](https://osu.ppy.sh/users/9560694)**, ![][flag_MY] [ShaneLiang](https://osu.ppy.sh/users/6716499) |
```

| **:seemsgood:** | ![][flag_MY] **[malaidan](https://osu.ppy.sh/users/14279913)**, ![][flag_MY] [haruchi](https://osu.ppy.sh/users/4845266) |

> | **1v2** | ![][flag_MY] **[KumaXione](https://osu.ppy.sh/users/2582531)**, ![][flag_MY] [wuhua](https://osu.ppy.sh/users/2932510) |

Reviewer: ```suggestion
| **1v2** | ![][flag_MY] **[Kumaxione](https://osu.ppy.sh/users/2582531)**, ![][flag_MY] [wuhua](https://osu.ppy.sh/users/2932510) |
```

| **:okniceperson:** | ![][flag_MY] **[milo Penguin](https://osu.ppy.sh/users/10702232)**, ![][flag_MY] [Maopo](https://osu.ppy.sh/users/15293336) |
| **pufferfishes** | ![][flag_MY] **[Watson Amelia](https://osu.ppy.sh/users/12674638)**, ![][flag_MY] [Mano Aloe](https://osu.ppy.sh/users/5429513) |
| **Rocket League Players** | ![][flag_MY] **[GKen](https://osu.ppy.sh/users/12633052)**, ![][flag_MY] [LavenderEclipse](https://osu.ppy.sh/users/13510754) |
| **Top 10 Anime Plot Twist** | ![][flag_MY] **[My Angel Petra](https://osu.ppy.sh/users/8691555)**, ![][flag_MY] [xArief](https://osu.ppy.sh/users/12694468) |
| **hidden sucks** | ![][flag_MY] **[zorrospg](https://osu.ppy.sh/users/18042234)**, ![][flag_MY] [727 not funny](https://osu.ppy.sh/users/14844663) |
| **"5 digit btw"** | ![][flag_MY] **[Zeph2003](https://osu.ppy.sh/users/10343292)**, ![][flag_MY] [Rampax](https://osu.ppy.sh/users/3995630) |
| **Communist LSD enjoyers** | ![][flag_MY] **[TequilaWolf](https://osu.ppy.sh/users/3633477)**, ![][flag_MY] [seabee](https://osu.ppy.sh/users/13472074) |

> | **hongjiapai trading co. ltd.** | ![][flag_MY] **[Ayumu Uehara](https://osu.ppy.sh/users/13626098)**, ![][flag_MY] [DaNoobSlayaar](https://osu.ppy.sh/users/8688777) |

Author: ok i will change this

| **Horny 14s** | ![][flag_MY] **[Snowy5441](https://osu.ppy.sh/users/13865320)**, ![][flag_MY] [AsianEnd](https://osu.ppy.sh/users/17010088) |
| **I was forced to play this** | ![][flag_MY] **[[Winter]](https://osu.ppy.sh/users/7373182)**, ![][flag_MY] [Fancyyy](https://osu.ppy.sh/users/12616268) |
| **Good Morning China I have Ice Cream now I Love Ice Cream** | ![][flag_MY] **[sealw4ll](https://osu.ppy.sh/users/12289818)**, ![][flag_MY] [MinaMyoui](https://osu.ppy.sh/users/6996926) |
| **The Alieppie Supremacy** | ![][flag_MY] **[Aleppie](https://osu.ppy.sh/users/766445)**, ![][flag_MY] [Alippie](https://osu.ppy.sh/users/13173302) |
| **low combo SS** | ![][flag_MY] **[vernonlim](https://osu.ppy.sh/users/10167542)**, ![][flag_MY] [cloud_chaser02](https://osu.ppy.sh/users/7673764) |
| **Mio Supremacy** | ![][flag_MY] **[Judicious](https://osu.ppy.sh/users/8087710)**, ![][flag_MY] [Changed](https://osu.ppy.sh/users/8009988) |
| **safe spot** | ![][flag_MY] **[Addam](https://osu.ppy.sh/users/11716599)**, ![][flag_MY] [Kiddthehaz](https://osu.ppy.sh/users/10838547) |

> | **chunchunmaru** | ![][flag_MY] **[MyAngelFable](https://osu.ppy.sh/users/11234664)**, ![][flag_MY] [_Rel_](https://osu.ppy.sh/users/11622287) |

Reviewer 3: ```suggestion
| **chunchunmaru** | ![][flag_MY] **[MyAngelFable](https://osu.ppy.sh/users/11234664)**, ![][flag_MY] [\_Rel\_](https://osu.ppy.sh/users/11622287) |
```

| **Sheep** | ![][flag_MY] **[FaithInOsu](https://osu.ppy.sh/users/10178187)**, ![][flag_MY] [Vexine](https://osu.ppy.sh/users/4042706) |
| **i just cooked cheese** | ![][flag_MY] **[RyzenPP](https://osu.ppy.sh/users/16391241)**, ![][flag_MY] [NAVolkan](https://osu.ppy.sh/users/14981515) |
| **Chicken Mutton** | ![][flag_MY] **[Inequality](https://osu.ppy.sh/users/8455034)**, ![][flag_MY] [Hiqal](https://osu.ppy.sh/users/13892584) |
| **Penguin Enjoyers** | ![][flag_MY] **[wantoon](https://osu.ppy.sh/users/16042959)**, ![][flag_MY] [CXC1](https://osu.ppy.sh/users/12004902) |
| **e** | ![][flag_MY] **[Eximious](https://osu.ppy.sh/users/19872191)**, ![][flag_MY] [_GRiD](https://osu.ppy.sh/users/20743023) |
| **pen el grande** | ![][flag_MY] **[Zervail](https://osu.ppy.sh/users/13047340)**, ![][flag_MY] [DARKWEBXD](https://osu.ppy.sh/users/13060849) |
| **amriourmommy** | ![][flag_MY] **[\[Fubuki\]](https://osu.ppy.sh/users/14411486)**, ![][flag_MY] [GideonTan](https://osu.ppy.sh/users/17426621) |
| **Monkeybsns** | ![][flag_MY] **[ClawViper](https://osu.ppy.sh/users/2681361)**, ![][flag_MY] [Jeffrey](https://osu.ppy.sh/users/4777134) |

> | **aidanyunusfanclub** | ![][flag_MY] **[pootisman1547](https://osu.ppy.sh/users/6611419)**, ![][flag_MY] [#RESTRICTED](https://osu.ppy.sh/users/18897374) |

Reviewer: hmm, is there any reason why the user here is censored? iirc tournament wiki pages are meant as a form of "documentation" of sorts so players who participate in the tournament but has since been restricted should still be included as is in the wiki

(see : the wiki pages of the earlier owc tournaments (e.g. [owc #1](http://osu.ppy.sh/wiki/en/Tournaments/OWC/1)) for reference)

  5. [Yu-Peng Chen @HOYO-MiX - Preserver of Fortune (Shurelia) \[77\]](https://osu.ppy.sh/beatmapsets/1370858#osu/2834500)
- Hidden
  1. [lasah - EXCERPTS FROM THE DIARIES OF _ (Reincarnation) \[None\]](https://osu.ppy.sh/beatmapsets/868684#osu/1815675)
  2. [Gesu no Kiwami Otome. - Romance ga Ariamaru (emilia) \[new try\]]()
- HardRock
  1. [Sagara Kokoro - Hoshizora no Ima (Reillia) \[Kaliren's Extra\]](https://osu.ppy.sh/beatmapsets/880854#osu/1955078)
  2. [SLAM & NATO - PARADOXX (zzx) \[Another\]](https://osu.ppy.sh/beatmapsets/1116086#osu/2331568)
- DoubleTime
  1. [Huang Ling - Deng Dai (Bellicose) \[AIR's Insane\]](https://osu.ppy.sh/beatmapsets/1094296#osu/2294800)
  2. [IOSYS - Chizome no Love Letter (Hellzero) \[Collab\]](https://osu.ppy.sh/beatmapsets/35878#osu/116095)
  3. [Akiyama Uni - Yuuga ni Sakase, Sumizome no Sakura ~ Border of Life (impossiblexu) \[Extra\]](https://osu.ppy.sh/beatmapsets/32380#osu/105965)
- FreeMod
  1. [Renard - TU4AR (P o M u T a) \[Extra\]](https://osu.ppy.sh/beatmapsets/82802#osu/228919)
  2. [Nakiri Ayame - Good-bye sengen (Deppyforce) \[Insane (tourney edit)\]](https://osu.ppy.sh/beatmapsets/715913#osu/1512691)

> -Tiebreaker

Reviewer 2: ```suggestion
- Tiebreaker
```

### Qualifiers

**[Download the mappack here! (66.9 MB)](https://cdn.discordapp.com/attachments/660200932981866547/878692561995300894/oBAM_Qualifier_Mappool.rar)**

- NoMod
  1. [SawanoHiroyuki[nZk]:Tielle & Gemie - sh0ut (Gala) \[Collab Extra\]](https://osu.ppy.sh/beatmapsets/641854#osu/1428482)
  2. [penoreri - Sulfur (Deca) \[Brimstone\]](https://osu.ppy.sh/beatmapsets/1268600#osu/2636464)
  3. [Endorfin. - Sakurairo Prism (Karen) \[Expert\]](https://osu.ppy.sh/beatmapsets/1104027#osu/2306852)
  4. [YUC'e - macaron moon (Nathan) \[amai\]](https://osu.ppy.sh/beatmapsets/641882#osu/1360970)
- Hidden
  1. [Yorushika - Yuunagi, Bou, Hana Madoi (Deca) \[coco's Extra\]](https://osu.ppy.sh/beatmapsets/1029377#osu/2156980)
  2. [Aki - star river (ShirohaMyMommy) \[Lunatic\]](https://osu.ppy.sh/beatmapsets/908990#osu/1896891)
- HardRock
  1. [Sangatsu no Phantasia - Pink Lemonade (Chirony) \[Expert\]](https://osu.ppy.sh/beatmapsets/941246#osu/1965652)
  2. [slythe - cyber imagination (Mismagius) \[Present™\]](https://osu.ppy.sh/beatmapsets/790282#osu/2138306)
- DoubleTime

>   1. [Hua Ling / Miao*Jiang / Yan Ning / kinsen - Let the Wind Tell You (Moecho) \[About the Sunsets, the Moonlights, and the Starry Skies\]](https://osu.ppy.sh/beatmapsets/1378997#osu/2849349)

Reviewer 3: ```suggestion
  1. [Hua Ling / Miao\*Jiang / Yan Ning / kinsen - Let the Wind Tell You (Moecho) \[About the Sunsets, the Moonlights, and the Starry Skies\]](https://osu.ppy.sh/beatmapsets/1378997#osu/2849349)
```

Saturday, 9 October 2021:

| Team 1 |  |  | Team 2 | Match link |
| --: | :-: | :-: | :-- | :-- |
| nomicstreamers | 3 | **7** | **"5 digit btw"** | [#1](https://osu.ppy.sh/mp/92140758)

Sunday, 10 October 2021:

| Team 1 |  |  | Team 2 | Match link |
| --: | :-: | :-: | :-- | :-- |
| low combo SS | 3 | **7** | **Ascendants** | [#1](https://osu.ppy.sh/community/matches/92202772)

Monday, 11 October 2021:

| Team 1 |  |  | Team 2 | Match link |
| --: | :-: | :-: | :-- | :-- |
| **Communist LSD enjoyers** | **7** | 6 | Ascendants | [#1](https://osu.ppy.sh/mp/92250750)

### Semifinals

Friday, 1 October 2021:

| Team 1 |  |  | Team 2 | Match link |
| --: | :-: | :-: | :-- | :-- |
| **Hutao Enjoyers** | **6** | 2 | ur mom | [#1](https://osu.ppy.sh/mp/91743361)
| **1v2** | **6** | 2 | Martin n willy show | [#1](https://osu.ppy.sh/mp/91745670)
| :seemsgood: | 4 | **6** | **Ascendants** | [#1](https://osu.ppy.sh/community/matches/91745843)

Saturday, 2 October 2021:

| Team 1 |  |  | Team 2 | Match link |
| --: | :-: | :-: | :-- | :-- |
| **nomicstreamers** | **6** | 0 | low combo SS | [#1](https://osu.ppy.sh/mp/91803867)

> | **cutest malaysia couple** | **6** | 0 |	:okniceperson: | [#1](https://osu.ppy.sh/community/matches/91806781)

Author: ok i will double check

Sunday, 3 October 2021:

| Team 1 |  |  | Team 2 | Match link |
| --: | :-: | :-: | :-- | :-- |
| **Ascendants** | **6** | 3 | cutest malaysia couple | [#1](https://osu.ppy.sh/community/matches/91865888)

Monday, 4 October 2021:

| Team 1 |  |  | Team 2 | Match link |
| --: | :-: | :-: | :-- | :-- |
| Hutao Enjoyers | 5 | **6** | **1v2** | [#1](https://osu.ppy.sh/community/matches/91908160)
| **"5 digit btw"** | **6** | 2 |	Communist LSD enjoyers | [#1](https://osu.ppy.sh/community/matches/91909065)

### Quarterfinals

Friday, 24 September 2021:

| Team 1 |  |  | Team 2 | Match link |
| --: | :-: | :-: | :-- | :-- |
| **nomicstreamers** | **6** | 1 | 1v2 | [#1](https://osu.ppy.sh/community/matches/91398903)
| **Martin n willy show** | **6** | 5 | Chicken Sandwich | [#1](https://osu.ppy.sh/community/matches/91402172)

Saturday, 25 September 2021:

| Team 1 |  |  | Team 2 | Match link |
| --: | :-: | :-: | :-- | :-- |
| **:okniceperson:** | **6** | 1 | Amamiya Kokoro | [#1](https://osu.ppy.sh/community/matches/91443708)
| **ur mom** | **6** | 1 | stwopid | [#1](https://osu.ppy.sh/community/matches/91450122)

> | **chunchunmaru Mio**** | **6** | 4 | Supremacy | [#1](https://osu.ppy.sh/community/matches/91454313)

Reviewer: ```suggestion
| **chunchunmaru** | **6** | 4 | Supremacy | [#1](https://osu.ppy.sh/community/matches/91454313)
```

Author: i really dint save before commiting (not like this)

| Team 1 |  |  | Team 2 | Match link |
| --: | :-: | :-: | :-- | :-- |
| **1v2** | **5** | 1 |	chunchunmaru | [#1](https://osu.ppy.sh/mp/91151060)
| **Communist LSD enjoyers** | **5** | 1 | ur mom | [#1](https://osu.ppy.sh/community/matches/91157843)
| **Hutao Enjoyers** | **5** | 4 | Monkeybsns | [#1](https://osu.ppy.sh/community/matches/91160656)
| The Alieppie Supremacy | 0 | **5** | **Mio Supremacy** | [#1](https://osu.ppy.sh/community/matches/91161963)
| **cutest malaysia couple** | **5** | 2 | 123ganggang | [#1](https://osu.ppy.sh/mp/91162071)
| **"5 digit btw"** | **5** | 0 | Martin n willy show | [#1](https://osu.ppy.sh/mp/91164633)
| **:seemsgood:** | **5** | 0 | pufferfishes | [#1](https://osu.ppy.sh/community/matches/91164653)
| Rocket League Players | 4 | **5** | **Chicken Sandwich** | [#1](https://osu.ppy.sh/community/matches/91167814)
| **low combo SS** | **5** | 2 | Ascendants | [#1](https://osu.ppy.sh/community/matches/91167941)

### Round of 32

Friday, 10 September 2021:

| Team 1 |  |  | Team 2 | Match link |
| --: | :-: | :-: | :-- | :-- |
| **cutest malaysia couple** | **5** | 1 | Like Father Like Son | [#1](https://osu.ppy.sh/mp/90671087) |

> | **nomicstreamers** | **5** | 0 | | The Maid of Orléans | [#1](https://osu.ppy.sh/community/matches/90674004) |

Reviewer: ```suggestion
| **nomicstreamers** | **5** | 0 | The Maid of Orléans | [#1](https://osu.ppy.sh/community/matches/90674004) |
```

   - Z Score is calculated PER MAP. Your Qualifiers standing will be determined by your team’s total Z Score on all 10 maps played, and will be sorted by ascending total Z Score, with the team with the highest total Z Score value placing 1st.
5. The 32 Teams with the **lowest amount of Z Score in total** will advance to the Round of 32 as follows:

| Qualifier ranking | Round of 32 bracket |
| :-: | :-- |
| #1 - #8 | **Top Seed** |
| #9 - #16 | **High Seed** |
| #17 - #24 | **Mid Seed** |
| #25 - #32 | **Low Seed** |

6. Failure to attend in any of the predetermined Qualifier lobbies will result in an instant elimination from the tournament.

### Match procedure

1. The match lobby will be created by the referee, and all four players will be invited.
2. Teams have a strict 15 minute grace period to have both players available to play the match. If 15 minutes pass and a team does not have enough players to play the match, the other team will receive a win by default.
3. The referee can give host to the players for warmups only. The referee will clear host for the after warmups. A team will select a map by naming the map in the multiplayer chat for the referee to mpcommand.

> 1. both captains will `!roll`. The higher number rolling captain gets first pick second ban.
> 2. Each Team is obligated to ban **two beatmaps** from the mappool, with the first ban given prior to the beginning of the match and the second ban given after 4 maps have been played. These beatmaps are not allowed to be picked by any player during the entire match.
> 4. In a FreeMod pick, Both players has to apply at least one mod to play the beatmap with. Allowed mods are EZ, HR, HD, FL, or any possible combinations of the four mods.
>    - Both teams need at least 1 __HR__ or 1 __HD__ (EZHD counts as using HD)
>    - EZ will have their own independent multiplier depending on the FM pick.
>    - There is no bonus multiplier for FL
>    - Playing a FreeMod pick without any mods activated is not allowed.
> 5. The results of each match and any other relevant information regarding the match will be posted on the Discord server after the match has been concluded by the responsible referees.

Reviewer 2: ```suggestion
4. Both captains will execute `!roll` command. The team that get the highest number will get to pick the first map and then ban after another team pick which map will be banned.
5. Each team is obligated to ban **two beatmaps** from the mappool, with the first ban given prior to the beginning of the match and the second ban given after 4 maps have been played. These beatmaps are not allowed to be picked by any player during the entire match.
6. In a FreeMod pick, Both players has to apply at least one mod to play the beatmap with. Allowed mods are EZ, HR, HD, FL, or any possible combinations of the four mods.
   - Both teams need at least 1 __HR__ or 1 __HD__ (EZHD counts as using HD)
   - EZ will have their own independent multiplier depending on the FM pick.
   - There is no bonus multiplier for FL
   - Playing a FreeMod pick without any mods activated is not allowed.
7. The results of each match and any other relevant information regarding the match will be posted on the Discord server after the match has been concluded by the responsible referees.
```

---
tags:
  - o!BAM
---

# osu! Battle Archive Malaysia

![o!BAM banner](img/banner.png) 

The **osu! Battle Archive Malaysia** (***o!BAM***) was a Malaysian double-elimination 2v2 osu! tournament hosted by ![][flag_MY] [LouisZpro](https://osu.ppy.sh/users/6231292) and ![][flag_MY] [Tzero](https://osu.ppy.sh/users/6088976). It was the first instalment of the osu! Battle Archive Malaysia.

## Tournament schedule

| Event | Timestamp |
| --: | :-- |
| Registration phase | 2021-08-10/2021-08-29 |
| Qualifiers | 2021-09-04/2021-09-05 |
| Live drawings | 2021-09-06 (13:00 UTC) |
| Round of 32 | 2021-09-11/2021-09-12 |
| Round of 16 | 2021-09-18/2021-09-19 |
| Quarterfinals | 2021-09-25/2021-09-26 |
| Semifinals | 2021-10-02/2021-10-03 |
| Finals (week 1) | 2021-10-09/2021-10-10 |
| Finals (week 2) | 2021-10-16/2021-10-17 |

## Prizes

| Placing | Prize(s) |
| :-: | :-- |

> | ![Gold crown](/wiki/shared/crown-gold.png "1st place") | 2 months of osu!supporter per team member, animated profile banners, exclusive wrist-rest custom-designed by [Wristlabs](https://wristlabs.net) |

Reviewer 3: The company name is Wrist Labs

```suggestion
| ![Gold crown](/wiki/shared/crown-gold.png "1st place") | 2 months of osu!supporter per team member, animated profile banners, exclusive wrist rest custom-designed by [Wrist Labs](https://wristlabs.net) |
```

| Administrator | ![][flag_MY] [Laphii](https://osu.ppy.sh/users/9364594), ![][flag_SG] [Ayucchi](https://osu.ppy.sh/users/7823498) |
| Mappool selector | ![][flag_MY] [LouisZpro](https://osu.ppy.sh/users/6231292), ![][flag_MY] [Tzero](https://osu.ppy.sh/users/6088976), ![][flag_SG] [Ayucchi](https://osu.ppy.sh/users/7823498) |
| Streamer | ![][flag_MY] [Laphii](https://osu.ppy.sh/users/9364594), ![][flag_MY] [Hand Sanitiser](https://osu.ppy.sh/users/5091293), ![][flag_MY] [Koyaiba](https://osu.ppy.sh/users/7062729) |
| Commentator | ![][flag_MY] [MyAngelFable](https://osu.ppy.sh/users/11234664), ![][flag_MY] [Malaidan](https://osu.ppy.sh/users/14279913), ![][flag_MY] [fancyyy](https://osu.ppy.sh/users/12616268), ![][flag_MY] [sealw4ll](https://osu.ppy.sh/users/12289818), ![][flag_MY] [Banner](https://osu.ppy.sh/users/14290988), ![][flag_MY] [MinaMyoui](https://osu.ppy.sh/users/6996926), ![][flag_MY] [IHateAutoNicks](https://osu.ppy.sh/users/10758397), ![][flag_MY] [Chiyuu](https://osu.ppy.sh/users/8226107), ![][flag_MY] [Its not Marvin](https://osu.ppy.sh/users/13600612) |

> | Referee | ![][flag_MY] [Xyrelyx](https://osu.ppy.sh/users/3765632), ![][flag_MY] [Laphii](https://osu.ppy.sh/users/9364594),![][flag_VN] [rock-on](https://osu.ppy.sh/users/9676089), ![][flag_MY] [Vexine](https://osu.ppy.sh/users/4042706), ![][flag_FR] [Taevas](https://osu.ppy.sh/users/7276846), ![][flag_MY] [DaNoobSlayaar](https://osu.ppy.sh/users/8688777), ![][flag_MY] [Wen4ng3L](https://osu.ppy.sh/users/9288833), ![][flag_MY] [Hand Sanitiser](https://osu.ppy.sh/users/5091293), ![][flag_MY] [MinaMyoui](https://osu.ppy.sh/users/6996926) |

Reviewer 3: ```suggestion
| Referee | ![][flag_MY] [Xyrelyx](https://osu.ppy.sh/users/3765632), ![][flag_MY] [Laphii](https://osu.ppy.sh/users/9364594), ![][flag_VN] [rock-on](https://osu.ppy.sh/users/9676089), ![][flag_MY] [Vexine](https://osu.ppy.sh/users/4042706), ![][flag_FR] [Taevas](https://osu.ppy.sh/users/7276846), ![][flag_MY] [DaNoobSlayaar](https://osu.ppy.sh/users/8688777), ![][flag_MY] [Wen4ng3L](https://osu.ppy.sh/users/9288833), ![][flag_MY] [Hand Sanitiser](https://osu.ppy.sh/users/5091293), ![][flag_MY] [MinaMyoui](https://osu.ppy.sh/users/6996926) |
```

| **:seemsgood:** | **5** | 0 | Ahmad | [#1](https://osu.ppy.sh/community/matches/90797103) |
| Amamiya Kokoro | 3 | **5** | **pufferfishes** | [#1](https://osu.ppy.sh/community/matches/90799380) |
| **1v2** | **5** | 0 | TUYU Enjoyers | [#1](https://osu.ppy.sh/community/matches/90802376) |
| **Communist LSD enjoyers** | **5** | 0 | The Alieppie Supremacy | [#1](https://osu.ppy.sh/community/matches/90802379) |

Monday, 13 September 2021:

| Team 1 |  |  | Team 2 | Match link |
| --: | :-: | :-: | :-- | :-- |
| 洪家派贸易有限公司(hongjiapai trading co. ltd) | 4 | **5** | **123ganggang** | [#1](https://osu.ppy.sh/community/matches/90850024) |

Tuesday, 14 September 2021:

| Team 1 |  |  | Team 2 | Match link |
| --: | :-: | :-: | :-- | :-- |
| **chunchunmaru** | **5** | 0 | stwopid | [#1](https://osu.ppy.sh/community/matches/90888112) |

Wednesday, 15 September 2021:

| Team 1 |  |  | Team 2 | Match link |
| --: | :-: | :-: | :-- | :-- |
| **Hutao Enjoyers** | **5** | 0 | Rocket League Players | [#1](https://osu.ppy.sh/community/matches/90942079) |

## Ruleset

### General Rules

1. osu! Battle Archive Malaysia uses double-elimination bracket and is a 2v2 osu! tournament.

> 2. Only players with the country flag of ![][flag_MY] Malaysia may participate.

Author: dont think there is other flags participants unless they changes it

11. Match schedules will be written in UTC+8.
12. There will be pickems available **internationally** for anyone to have a chance at earning a prize, even if they aren't local.

### Qualifiers Rules

1. Each team captain will have to sign up to one of the Qualifier lobbies that has been scheduled and prepared by the tournament management in advance.
2. In the lobby, all teams will have to consecutively play all of the ten Qualifier beatmaps in the order of NM1 -> NM2 -> NM3 -> NM4 -> HD1 -> HD2 -> HR1 -> HR2 -> DT1 -> DT2.
3. Teams **are not allowed** to join in (or register for) more than one Qualifier lobby.
4. Based on their performance in the Qualifier, players will be ranked based on the amount of Z-Score they accumulate.
   - Z-Score compares your team score relative to the average team score achieved in a map and assigns a value to it. Z-Score is measured in terms of the standard deviation of the team score from the mean team score achieved within that map. If your value is higher than 0, it means that your total team score is more than the average team score on that specific map, and vice versa with values lower than 0.

>    - Z-Score is calculated PER MAP. Your Qualifiers standing will be determined by your team's total Z-Score on all 10 maps played, and will be sorted by ascending total Z-score, with the team with the highest total Z-Score value placing 1st.

Reviewer 3: woops that's my bad

```suggestion
   - Z-Score is calculated PER MAP. Your Qualifiers standing will be determined by your team's total Z-Score on all 10 maps played, and will be sorted by ascending total Z-Score, with the team with the highest total Z-Score value placing 1st.
```

> 5. The 32 Teams with the **highest amount of Z-Score in total** will advance to the Round of 32 as follows:

Reviewer 3: ```suggestion
5. The 32 teams with the **highest amount of Z-Score in total** will advance to the Round of 32 as follows:
```

### Match procedure

1. The match lobby will be created by the referee, and all four players will be invited.
2. Teams have a strict 15 minute grace period to have both players available to play the match. If 15 minutes pass and a team does not have enough players to play the match, the other team will receive a win by default.
3. The referee can give host to the players for warmups only. The referee will clear host for the after warmups. A team will select a map by naming the map in the multiplayer chat for the referee to mpcommand.
4. Both captains will execute `!roll` command. The team that get the highest number will get to pick the first map and then ban after another team pick which map will be banned.
5. Each team is obligated to ban **two beatmaps** from the mappool, with the first ban given prior to the beginning of the match and the second ban given after 4 maps have been played. These beatmaps are not allowed to be picked by any player during the entire match.
6. In a FreeMod pick, Both players has to apply at least one mod to play the beatmap with. Allowed mods are EZ, HR, HD, FL, or any possible combinations of the four mods.

>    - Both teams need at least 1 `HR` or 1 `HD` (EZHD counts as using HD)

Reviewer 3: use bolding if you want to emphasise this

```suggestion
   - Both teams need at least 1 **HR** or 1 **HD** (EZHD counts as using HD)
```

>    - Both teams need at least 1 **HR** or 1 **HD** (EZHD counts as using HD)

Reviewer 3: ```suggestion
   - Both teams need at least 1 **HR** or 1 **HD** (EZHD counts as using HD).
```
